php/php.latte/nbproject/project.xml

```diff
@@ -147,7 +147,7 @@
                     <compile-dependency/>
                     <run-dependency>
                         <release-version>2</release-version>
-                        <specification-version>2.19</specification-version>
+                        <specification-version>2.66</specification-version>
```

Comment:
For @junichi11 to review, but likely OK.

Reply:
I suppose it's OK.